watertap/unit_models/electrodialysis_0D.py

```diff
@@ -411,7 +504,18 @@ def build(self):
             units=pyunits.dimensionless,
             doc="water recovery ratio calculated by mass",
         )
-        # TODO: consider adding more performance as needed.
+        self.velocity_diluate = Var(
```

Comment:
Here we are setting the precedent for non-indexed but channel specific properties that don't exist on the state block, do we think this implementation is best? Options are
1. self.velocity_diluate
2. self.diluate.veloctity
3. self.velocity["diluate"]

which all are doable, just will have different nuances

Reply:
I would leave as the current setting to be consistent with 1d file.  I don't see drawback of this setup and it provides convenient use at the fs level.